src/test/java/uk/gov/hmcts/reform/sandl/snlevents/controllers/HearingPartControllerTest.java

```diff
@@ -105,6 +105,19 @@ public void upsertHearingPart_savesHearingPartToService() throws Exception {
         assertThat(response).isEqualToComparingFieldByFieldRecursively(createHearingPart());
     }
 
+    @Test
+    public void deleteHearingPart_deletesHearingPart() throws Exception {
+        val id = UUID.randomUUID();
+
+        val serviceResult = createHearingPart();
+        serviceResult.setDeleted(true);
+        when(hearingPartService.deleteHearingPart(id)).thenReturn(serviceResult);
+
+        val response = mvc.deleteAndReturnResponse(URL + "/" + id);
+
```

Comment:
can we not check response code as well for this deletion.?  i don't know whether we do same for other controller test.

Reply:
status code is checked in mvc.deleteAndReturnResponse(URL + "/" + id)